lua/nvim-treesitter/parsers.lua

```diff
@@ -1512,7 +1512,7 @@ list.t32 = {
   install_info = {
     url = "https://codeberg.org/xasc/tree-sitter-t32",
```

Comment:
I can see that there exists a GitLab mirror for the language now. Very quick and clean work 😄 

I think it's fine to change the url to point directly to the GitLab mirror now

Reply:
and, more importantly, revert my last two commits